spec/controllers/users_controller_spec.rb

```diff
@@ -84,15 +88,19 @@
       login_as(u)
       it "renders successfully" do
         get :show, params: {id: u.id}
-        expect(response).to be_success
-        expect(response.body).to match(/Showing user/m)
+        expect(response).to be_successful
+        expect(response.body).to match(/Contact/m)
+        expect(response.body).to match(/About/m)
+        expect(response.body).to match(/Courses/m)
+        expect(response.body).to match(/Private Settings/m)
+        expect(response.body).to match(/API Settings/m)
```

Comment:
This is doing a string match on the page right?

Reply:
Yup!